plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/testdsl/TestAnnotations.kt

```diff
@@ -25,7 +29,6 @@ object TestVersions {
 @Retention(AnnotationRetention.RUNTIME)
 annotation class GradleTestVersions(
     val minVersion: String = TestVersions.minSupportedGradleVersion,
-    val maxVersion: String = TestVersions.maxSupportedGradleVersion,
```

Comment:
we were never overriding this anyway, and making it an annotation property was preventing me from making it dynamic based Java version